.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
+
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v2.4.0
+      hooks:
+      -   id: trailing-whitespace
+      -   id: end-of-file-fixer
+      -   id: check-docstring-first
+      -   id: check-json
+      -   id: check-yaml
+      -   id: double-quote-string-fixer
+
+  -   repo: https://github.com/ambv/black
+      rev: 19.10b0
+      hooks:
+      - id: black
+
+
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v2.4.0
+      hooks:
+      - id: flake8
+
+  # -   repo: https://github.com/asottile/seed-isort-config
+  #     rev: v1.9.4
+  #     hooks:
+  #     -   id: seed-isort-config
+  # -   repo: https://github.com/pre-commit/mirrors-isort
+  #     rev: v4.3.21
+  #     hooks:
+  #     -   id: isort
```

Comment:
@andersy005 - do we want to uncomment this?

Reply:
Fixed!